internal/test/faas/awslambda/mongodb/go.mod

```diff
@@ -0,0 +1,26 @@
+module go.mongodb.go/mongo-driver/internal/test/mongodb
+
+go 1.19
+
+replace go.mongodb.org/mongo-driver => ../../../../../
+
+require (
+	github.com/aws/aws-lambda-go v1.36.1
```

Comment:
This version of the AWS Lambda library is from Dec 2022. Is there a reason not to use the latest version (v1.41.0)?